src/plugins/intel_gpu/tests/fusions/gemm_fusion_test.cpp

```diff
@@ -388,7 +394,7 @@ INSTANTIATE_TEST_SUITE_P(fusings_gpu, gemm_2in_act_scale_eltwise, ::testing::Val
     gemm_test_params{ CASE_GEMM_ELTWISE_2IN_FP16_1, 3, 6 },
     gemm_test_params{ CASE_GEMM_ELTWISE_2IN_U8S8_1, 3, 6 },
     gemm_test_params{ CASE_GEMM_ELTWISE_2IN_S8U8_1, 3, 6 },
-    gemm_test_params{ CASE_GEMM_ELTWISE_2IN_U8S8_2, 3, 3 , "gemm_mmad_int8" },
+    gemm_test_params{ CASE_GEMM_ELTWISE_2IN_U8S8_2, 3, 3, "gemm_mmad_int8" },
     // gemm_test_params{ CASE_GEMM_ELTWISE_2IN_U8S8_2, 3, 3 , "gemm_mmad_int8_slm" },   // tolerance issue
-    gemm_test_params{ CASE_GEMM_ELTWISE_2IN_FP16_2, 3, 3 , "gemm_tiled_opt" },
+    gemm_test_params{ CASE_GEMM_ELTWISE_2IN_FP16_2, 3, 3, "gemm_tiled_opt" },
```

Comment:
I think this is a real issue, not a test issue. Let's not fix it in this PR.

Reply:
Reverted related changes.